syntax = "proto3";
package communication_objects;

enum EnvironmentCommand {
    STEP = 0;
    RESET = 1;
    CLOSE = 2;
}